src/libaudqt/eq-preset-qt.cc

```diff
@@ -260,6 +260,7 @@ static void show_import_dialog(QWidget * parent, PresetView * view,
     dialog->setFileMode(QFileDialog::ExistingFile);
     dialog->setLabelText(QFileDialog::Accept, _("Load"));
     dialog->setNameFilter(_(name_filter));
+    dialog->setWindowRole("filedialog");
```

Comment:
Maybe `file-dialog` to be consistent with e.g. `color-dialog`?

Reply:
+1